OneCommunityGlobal/HGNRest
# Lucile - add work confirmation improvements to people report

###Description
9. (PRIORITY MEDIUM) Jae: Add work confirmation improvements to People Report (WIP Lucile)
Admin or Owner login → Reports → Reports → People → Choose
Remove “hours logged this week” section: If a user is INACTIVE (has an end date), then the “Hours Logged This Week” square should not show. Center the remaining ones.
Please also add the User’s Title under their name.

THIS PART IS THE ONE NOT FINISHED YET:
Please also add a “Rehireable” section under their name that can be clicked as Yes or No


###Related PR
To test this backend PR you need to checkout the #836 frontend PR.
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/836)

###Mainly changes explained:
1 - isRehireable was added to UserProfile Model
2 - isRehireable management was added in userProfileController

###Before (sorry for my slow connexion)

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/111132148/e92b0a4c-9f38-4129-963b-645cdada4b9d

###After (sorry for my slow connexion)



https://github.com/OneCommunityGlobal/HGNRest/assets/111132148/1587e394-e643-4f06-9c1c-9b6ade8e35f3



###How to test:
Git checkout Lucile_add_work_confirmation_improvements_to_people_report in the backend
do npm install and npm run build to run this PR locally

Git checkout Lucile_add_work_confirmation_improvements_to_people_report in the frontend
do npm install and npm run start:local 

Admin or Owner login → Reports → Reports → People → Choose someone with an end date 
You should be able to checkout and checkout the rehireable check box on the right.
The chosen check should remain even if the page is refreshed.

Admin or Owner login → Reports → Reports → People → Choose someone without an end date 
You should not see the rehireable checkbox
